pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -41,16 +38,16 @@ Base.setproperty!(x::MPtr, f::Symbol, v) = Globals.ASS_REC(x, RNamObj(f), v)
 Base.setproperty!(x::MPtr, f::Union{AbstractString,Int64}, v) = Globals.ASS_REC(x, RNamObj(f), v)
 
 #
-Base.zero(x::GAPInputType_internal) = Globals.ZERO(x)
-Base.one(x::GAPInputType_internal) = Globals.ONE(x)
-Base.:-(x::GAPInputType_internal) = Globals.AINV(x)
+Base.zero(x::Union{MPtr,FFE}) = Globals.ZERO(x)
+Base.one(x::Union{MPtr,FFE}) = Globals.ONE(x)
+Base.:-(x::Union{MPtr,FFE}) = Globals.AINV(x)
```

Comment:
It's immediately clear now that these functions only apply to MPtr and FFE, nothing else.